spec/fixtures/1846/12666.json

```diff
@@ -1201,8 +1201,8 @@
     },
     {
       "type": "lay_tile",
-      "entity": "GT",
-      "entity_type": "corporation",
+      "entity": "MC",
+      "entity_type": "company",
```

Comment:
This is how current game will fail. Pass done in the track step by the corporation, rather than special_track.